aidants_connect_common/templates/ie-deprecation.html

```diff
@@ -0,0 +1,34 @@
+{% load static ac_common %}
+
+<template id="ie-end-of-life-tpl">
+  <div class="ie-end-of-life">
+    <h1>Attention ! Il semble que vous utilisiez IE11</h1>
+    <p>
+      Le 15 juin dernier
+      <a href="https://blogs.windows.com/windowsexperience/2022/06/15/internet-explorer-11-has-retired-and-is-officially-out-of-support-what-you-need-to-know/">
+        Microsoft a offciellement annoncé la fin de vie d'Internet Explorer au profit de Edge (lien en anglais)
+      </a>.
+      Internet Explorer ne reçoit plus de mise à jour de sécurité depuis ce jour et n'est plus considéré comme un
+      navigateur sûr pour Aidants Connect.
+    </p>
+    <p>
+      C'est pourquoi l'application Aidant Connect ne supportera plus Internet Explorer à compter du 31 mars 2022.
```

Comment:
```suggestion
      C'est pourquoi, à compter du 31 mars 2022, nous ne pourrons plus garantir le bon fonctionnement de l'application Aidant Connect avec le navigateur Internet Explorer.
```

Pour éviter l'anglicisme "supporter" qui est plutôt supporté par un public initié.